src/transformers/models/paligemma/modeling_paligemma.py

```diff
@@ -98,6 +100,38 @@ def forward(self, image_features):
         return hidden_states
 
 
+def token_type_ids_mask_function(
```

Comment:
all the diff in this file is copy/paste, except for the removal of the `tokens_per_image` (unused arg) in this function